### DISCUSSION TITLE
report generator like crystal

### DISCUSSION THREAD

**Discussion by isbincApps** [supabase/supabase#12609] [Feature Requests]
Hello, how can I generate reports like the ones generated by the crystal report

**Comment by barbinbrad** [comment]
You can connect Crystal Report to your Postgres database: https://wiki.postgresql.org/wiki/PostgreSQL_and_Crystal_Reports

I'm not very familiar with this space, but after a quick Googling, there seem to be some paid solutions like Holistics, Looker, Mode, Redash, and Metabase. Not necessarily pdf-first, but that's probably a good thing. There's also an open source BI tool called [lightdash](https://github.com/lightdash/lightdash) that looks cool.

**Reply by isbincApps** [reply]
hi, how to connect supabase postgres db to some one report generator, do you have any example

**Comment by isbincApps** [comment]
hi, how to connect supabase postgres db to some one report generator, do you have any example
